recipes/gtsam/all/conanfile.py

```diff
@@ -0,0 +1,135 @@
+import os
+import stat
+import shutil
+from conans import ConanFile, tools, CMake
+
+class gtsamConan(ConanFile):
+    name = "gtsam"
+    license = "BSD-2-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

https://github.com/borglab/gtsam/blob/develop/LICENSE.BSD